### ISSUE TITLE
Color not change

### ISSUE THREAD

**Issue by Arumugamsix** [google/model-viewer#3805]
We have implemented Color change functionality. its working fine on yesterday but today i have checked i have experience below error
**Uncaught TypeError: material.pbrMetallicRoughness[channel].texture.source.setURI is not a function** 

![image](https://user-images.githubusercontent.com/73950837/190079208-b2026e65-9009-4da9-a435-fb5b00311a5d.png)

**Comment by elalish** [comment]
Indeed, this is from the [v2.0](https://github.com/google/model-viewer/releases/tag/v2.0.0) upgrade. You may want to pin to an earlier version range until you have a chance to update to `createTexture`, though that shouldn't be too hard. You can follow the [examples](https://modelviewer.dev/examples/scenegraph/#createTexturesExample) here.
